File: android/app/src/main/java/jagdx/IDirect3DSurface.java
package jagdx;

import jaclib.peer.IUnknown;
import jaclib.peer.ti;

public final class IDirect3DSurface extends IUnknown {

    public IDirect3DSurface(ti arg0) {
        super(arg0);
    }

    public int LockRect(int arg0, int arg1, int arg2, int arg3, int arg4, PixelBuffer arg5) { return 0; }

    public boolean UnlockRect() { return false; }
}

File: src/commonMain/kotlin/jagdx/IDirect3DSurface.kt
package jagdx

import jaclib.peer.IUnknown
import jaclib.peer.hb

expect class IDirect3DSurface(arg0: hb?) : IUnknown {
    fun LockRect(arg0: Int, arg1: Int, arg2: Int, arg3: Int, arg4: Int, arg5: PixelBuffer?): Int

    fun UnlockRect(): Boolean
}
